nimble/host/src/ble_hs_stop.c

```diff
@@ -33,6 +36,11 @@ static struct ble_gap_event_listener ble_hs_stop_gap_listener;
 SLIST_HEAD(ble_hs_stop_listener_slist, ble_hs_stop_listener);
 static struct ble_hs_stop_listener_slist ble_hs_stop_listeners;
 
+/* Track number of connections */
+static uint8_t conn_cnt;
+
+static struct ble_npl_callout terminate_tmo;
+
```

Comment:
I think these two globals (`conn_cnt`, `terminate_tmo`) should have the `ble_hs_stop_` prefix that the other symbols have.

Reply:
of course. thx